runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/status/DebugCapture.java

```diff
@@ -114,17 +114,22 @@ public Manager(DataflowWorkerHarnessOptions options, Collection<Capturable> capt
         return;
       }
 
+      JsonFactory datOtherFactory = new JacksonFactory();
```

Comment:
?

Reply:
Ugh.  No, I had added that briefly when testing and completely missed it when I cleaned up.  Removing now.